packages/react-components/react-list-preview/src/components/List/useListContextValues.ts

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+import { ListContextValues, ListState } from './List.types';
+
+export function useListContextValues_unstable(state: ListState): ListContextValues {
+  const { focusableItems, selection, as } = state;
+
+  const listContext = React.useMemo(
+    () => ({
+      focusableItems,
+      selection,
```

Comment:
There's no need for a memo here since you're using the context selector package - it's up to the context consumers to select smth that is relatively stable